drivers/ma_yya_report/app/models/ma_yya_report/report.rb

```diff
@@ -259,6 +261,14 @@ def label(key)
       end
     end
 
+    private def lgbtq_query
+      # Report defines LGBTQ as:
+      #   SexualOrientiation = gay(2), lesbian(3), bisexual(4), questioning(5), OR
+      #   Gender = transgender(5)
+      # Is it intentional that "other" SOs, and NB gender, are not considered LGBTQ for the report?
```

Comment:
Wondering if there is background on this, or we can clarify. I didn't make any change to the logic as it was defined for `G3a`.